src/__test__/test-utils/jestShim.js

```diff
@@ -0,0 +1,6 @@
+import {
```

Comment:
Can you add a comment explaining why we need this?

Reply:
I deleted this file -- I originally added it to fix a build error, but now I tried building the UI without it and it works.

Reply:
I had to actually re-add the file. It is needed because of this module: `'@wojtekmaj/enzyme-adapter-react-17'`. I have added a comment. The change is in my next branch